### ISSUE TITLE
should we turn the text ...

### ISSUE THREAD

**Issue by burner** [thewilsonator/Dlang-AGM#8]
... in the repo into github issues.
IMHO I think this help us keep track of the vision and evolve it over time.
I mean, we're using github to develop software, and then use "just" text to develop a vision (aka. software in a natural language) and don't use any of our awesome tools.

**Comment by thewilsonator** [comment]
Thats a great Idea. We can have an AGM (and probably also a DLF quarterly and Corporate Users Group) tag. This should probably be (re)done as an official dlang repo (cc @atilaneves). 

That reminds me, we ought to distill the discussion of the AGM into a document while its still fresh (cc @mdparker @ghost91-).

It'll be a while before I get home and am with it enough to do this so don't let me stop you doing great things.

**Comment by burner** [comment]
pretty much done already https://github.com/dlang/projects

**Comment by thewilsonator** [comment]
There's also a bunch of ideas, albeit less structured/concrete, over at https://github.com/truedat101/dlang-sv-community/issues that may be with looking at. Particularly the actionable ones.

**Comment by atilaneves** [comment]
The projects repo should be enough, no?

**Comment by thewilsonator** [comment]
Yes, I'm jut pointing out that that repo exists for those who may be unaware of it.
